src/lib/web-rpde-query.js

```diff
@@ -0,0 +1,81 @@
+import Settings from './settings.js';
+import { database_pool } from './database.js';
+
+
+
+
+class rpde_query {
+  constructor(afterTimestamp, afterId, limit) {
+    this.afterTimestamp = parseInt(afterTimestamp) > 0 ? parseInt(afterTimestamp) : null;
+    this.afterId = afterId;
+    this.limit = parseInt(limit) > 0 ? parseInt(limit) : Settings.rpdeDefaultLimit;
+    this.nextAfterTimestamp = null;
+    this.nextAfterId = null;
+    this.rows = []
+    this.dontServeDataUntilSecondsOld = Settings.rpdeDontServeDataUntilSecondsOld;
+  }
+
```

Comment:
let's either switch to camelCase everywhere or use lower case and underscores

Reply:
I'll change some but some specifically follow GET params set by the RPDE standard, so I'd like to keep those the same. Also, we seem to have established camelcase convention in settings so lets leave that and fix elsewhere if we want.